subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -690,12 +697,11 @@ int nct_connect(void)
 	err = getaddrinfo(NRF_CLOUD_HOSTNAME, NULL, &hints, &result);
 	if (err) {
 		LOG_DBG("getaddrinfo failed %d", err);
-
-		return err;
+		return -ECHILD;
```

Comment:
I think there are clearer codes for this, how about `ENETUNREACH` or `ENETDOWN`?

Reply:
This was to ensure accuracy in the error code translation.
I used `ECHILD` because that seemed unlikely to be returned by mqtt (socket) connect.
I agree, `ENETUNREACH` and `ENETDOWN` are clearer.  I'm fine with changing it if there is no concern for ambiguity.